src/app/view/components/_common/pattern-library/pattern-library.ts

```diff
@@ -34,6 +36,28 @@ export class PatternLibrary {
         this.cities.push({label: 'Paris', value: {id: 5, name: 'Paris', code: 'PRS'}});
     }
 
+    ngOnInit(): any {
+
+        // Fake data for datatable
+        this.cars = [
```

Comment:
Rename this var to some like ```dataTableDummyData``` or something more related with the component where we are using it.